conf/etc/iem_ha.json

```diff
@@ -9,5 +9,17 @@
             "SearchKey": "member"
         }
     },
-    "resource": {}
-}
+    "resource": {
+        "resources": ["io_path_health-c1", "io_path_health-c2"],
+        "modules": {
+            "IOStop": {
+                "SearchKey": "stop",
+                "IEM": "IEC:WS0080030002:On $host Detected unhealthy IO services. Stopping IO services on $host server."
```

Comment:
IO services on $host can not run due to either fault in hardware or a failed service. Services are being stopped and are most likely move to other node.

Reply:
Done